### ISSUE TITLE
It doesn`t work on vs2015

### ISSUE THREAD

**Issue by NanaTinyn** [LiamMorrow/Visual-Studio-Power-Mode#42]

**Comment by NanaTinyn** [comment]
Do you have the old version Installation package？If you have, can you send to me 744340810@qq.com

Please forgive my very badly English.

**Comment by NanaTinyn** [comment]
Thank you very much.

**Comment by tyh123ooo** [comment]
Do you Receive old version？I cannot open it on 2015,either
I install it,but it doesn't work on 2015

**Comment by BbChip0103** [comment]
I have same issue. I need old version, too.
bbchip13@gmail.com plz

**Comment by ipepta** [comment]
@NanaTinyn  @tyh123ooo  @BbChip0103 
compiled files attached in https://github.com/LiamMorrow/Visual-Studio-Power-Mode/issues/41
i'm using this version myself on vs 2015 and it's working very well.

**Comment by NanaTinyn** [comment]
[PowerMode (vs2015打字特效插件).zip](https://github.com/LiamMorrow/Visual-Studio-Power-Mode/files/1137424/PowerMode.vs2015.zip)
Here is the old version, I found it yesterday.

**Comment by tyh123ooo** [comment]
@NanaTinyn  thx,I will try

**Comment by tyh123ooo** [comment]
I revert to the Feb 9.2016 Version,but the references doesn't work,do u have the same problem?

**Comment by NanaTinyn** [comment]
No, it work on my vs2015,  please 'First uninstall after installation' , then open the PowerMode. And close the Automatic updates.

**Comment by tyh123ooo** [comment]
it works.I wanna see the original code.but that version cannot open

**Comment by NanaTinyn** [comment]
I am sorry that I cant help you, I dont know how did it worked.

**Comment by BbChip0103** [comment]
@NanaTinyn thank you. it works. your my hero right now.

**Comment by BbChip0103** [comment]
thank you. it works. your my hero right now.

**Comment by BbChip0103** [comment]
thank you. it works. your my hero right now.

**Comment by kidc2458** [comment]
> No, it work on my vs2015, please 'First uninstall after installation' , then open the PowerMode. And close the Automatic updates.

Hello. I am using VS 2015, and I found here because Power mode not work in VS 2015.

I downloaded your old version. but it did not work. 
so, What's mean " First uninstall after installation"?
